src/main/java/org/thoughtcrime/securesms/ApplicationContext.java

```diff
@@ -168,6 +168,9 @@ public void onCreate() {
               }
             }
 
+            // 2025.12.07: revert to enabled since the setting was removed
```

Comment:
```suggestion
            // 2025-12-16: The setting was removed.
            // Revert it to the default if it was changed in the past.
```

in order to follow ISO 8601 (which we're mostly using when writing dates in the code).

Also, I find the comment more understandable like this.